### PR TITLE
[WIP] Update ais_nginx.conf

### PR DESCRIPTION
**Pull request by apals**
Please notice that server_name has to be different for live and staging environment. Any ideas on how to make this automatically magic?

### CODE REVIEW

File: ais_nginx.conf [hunk 1]
```diff
@@ -9,13 +9,26 @@ server {
...
+    server_name ais2.armada.nu; # substitute your machine's IP address or FQDN
     charset     utf-8;
+    return 301 https://$server_name$request_uri;
+}
+
+server {
 
+    listen 443 ssl default_server;
+    listen [::]:443 ssl default_server;
+    include snippets/ssl-ais2.armada.nu.conf;
```
Comment (apals) [inline ais_nginx.conf]: name is wrong on live

File: ais_nginx.conf [hunk 2]
```diff
@@ -9,13 +9,26 @@ server {
     # the port your site will be served on
     listen      80;
     # the domain name it will serve for
-    server_name niwen2.aramada.nu; # substitute your machine's IP address or FQDN
+    server_name ais2.armada.nu; # substitute your machine's IP address or FQDN
```
Comment (apals) [inline ais_nginx.conf]: different name on live

Comment (walaro) [inline ais_nginx.conf]: server_name {{server_name}};
and then sed in the git/hooks/post-receive is a simple solution

Comment (apals) [inline ais_nginx.conf]: can u help me mr jw

Comment (walaro) [inline ais_nginx.conf]: I've put it in the push script for ais2 now, if it works, ill put it in ais aswell

**Pull request merged**